slam-libraries/viam-orb-slam3/setup_orbslam_macos.sh

```diff
@@ -0,0 +1,8 @@
+echo "Installing ORB_SLAM3 external dependencies"
+brew tap viamrobotics/brews
+brew install cmake glew opencv@4 eigen boost openssl
+LDFLAGS=-L/usr/local/opt/openssl@3/lib
+CPPFLAGS=-I/usr/local/opt/openssl@3/include
+PKG_CONFIG_PATH=/usr/local/opt/openssl@3/lib/pkgconfig
+brew install pangolin
+sudo ln -s /opt/homebrew/Cellar/pangolin/0.8/lib/* /usr/local/lib/ || true
```

Comment:
Without this, `orb_grpc_server` compiles but fails at runtime with a linking error, since the libraries for pangolin can't find each other. I'm not sure if this is a problem with how pangolin is built or with our brew package for it.